proto/span.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+package span;
+
+enum Event {
+    Unknown = 0;
+    CoprRequest = 1;
+    Scheduled = 2;
+    Snapshot = 3;
+    HandleRequest = 4;
+    HandleUnaryFixture = 5;
+    HandleChecksum = 6;
+    HandleDag = 7;
+    HandleBatchDag = 8;
+    HandleAnalyze = 9;
+    HandleCached = 10;
```

Comment:
I think these event names needs further discussion. For now, I prefer to reserve these simpliest ones:

```
TiKVCoprHandleRequest
TiKVCoprScheduleTask
TiKVCoprGetSnapshot
TiKVCoprExecuteDAGRunner
TiKVCoprExecuteBatchDAGRunner
```

Reply:
I think these events should be put inside tipb repos. There is no such `Dag` or `BatchDag` inside TiKV level. It's more tidb-related.